### PR TITLE
Trending topics

### PR DESCRIPTION
**Pull request by Ebenezr**
Trending topics component, done

### CODE REVIEW

File: src/components/ArticlelCards.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const ArticlelCards = () => {
```
Comment (nyandika) [inline src/components/ArticlelCards.tsx:1]: typo

File: src/components/Articles/ArticlelCards.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import moment from 'moment';
+import { articles } from './data';
+
+const ArticlelCards = () => {
```
Comment (nyandika) [inline src/components/Articles/ArticlelCards.tsx:4]: why do we have 2 similarly named components?

**Review by nyandika — CHANGES_REQUESTED** [review]
- Place the dummy data in a data folder in the root, and then call it from there.
- To further move it as close to the live experience place it in a json file and then serve it via http. something like [json-server](https://www.npmjs.com/package/json-server)
- Create an interface/type of data and then use this to statically type your fields
- I think you have to restructure this components a bit more

**Comment by Ebenezr** [comment]
noted, then on server fetching data any preference state manager or just a simple axios fetch

**Comment by nyandika** [comment]
> noted, then on server fetching data any preference state manager or just a simple axios fetch

axios is fine

**Pull request closed without merging**